daemon/proto/daemon.proto

```diff
@@ -64,9 +65,15 @@ message CreateEventRequest {
   repeated string exercises = 4;
   int32 available = 5;
   int32 capacity = 6;
-  string finishTime = 7;
+  string startTime = 7;
+  string finishTime = 8;
+  bool isBooked = 9;
```

Comment:
I am not sure where `isBooked` is used or set. I am not using it in the WebClient and seems it is not used in the Client either. in this case it can be deleted.

Reply:
true, it can be deleted.

Reply:
I was thinking to make something else at that point that's why I added. I guess :)